nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -208,21 +208,22 @@ let
     mkAfter = map (x: "docker-${x}.service") container.dependsOn;
   in rec {
     wantedBy = [] ++ optional (container.autoStart) "multi-user.target";
-    after = [ "docker.service" "docker.socket" ] ++ mkAfter;
+    after = mkAfter;
     requires = after;
```

Comment:
These lines doesn't make sense, it's better to drop them.
```
    after = mkAfter;
    requires = after;
```

Reply:
Sure it makes sense. These options are populated by the container `dependsOn` option.

Reply:
Yeah, `mkAfter` name is confusing.

Reply:
I agree with that. I've renamed the variable to `dependsOn` and removed the recursive attrset.